app/jsx/components/BreadcrumbComp.jsx

```diff
@@ -5,21 +5,24 @@ import { Link } from 'react-router'
 
 class BreadcrumbComp extends React.Component {
   render() {
-    var lastN = this.props.array.length - 1
-
-    var nodes = this.props.array.map(function(node, i) {
-      if (node.id == 'root') {
-        return (
-          <Link className={(i==lastN) ? "c-breadcrumb-link--active": null} to="/" key={i}>{node.name}</Link>
-        )
-      }
-      return (
-        <Link className={(i==lastN) ? "c-breadcrumb-link--active": null} to={"/unit/" + node.id} key={i}>{node.name}</Link>
-      )
-    });
+    let missingNavStyle = this.props.array ? null : { margin: '20px 0px 20px' },
```

Comment:
Items with no unit parent will no display a NavBarComp. Yet, the NavBarComp includes a bottom margin of 20px. This situation also begs whether we even need to display a breadcrumb. Either way, thing look ugly without me hardcoding this style into the breadcrumb for this use case and we should probably need to bring UX into the discussion eventually.